.travis.yml

```diff
@@ -20,6 +19,27 @@ install:
   - rustup target add riscv32imc-unknown-none-elf
   - rustup component add rustfmt
   - rustup component add clippy
+  - cargo install -f elf2tab --version 0.4.0
+  # Ubuntu doesn't package QEMU for RISC-V, so let's build it
+  - wget https://download.qemu.org/qemu-4.2.0.tar.xz
```

Comment:
Is there a way to get artifacts for qemu somehow?

Reply:
Yep, it actually is packaged in Debian, I just couldn't find it.